docs/ux-mockups/p2-3-shared-dashboard-baseline/decision.md

```
@@ -0,0 +1,8 @@
+# Shared Dashboard UX decision
+
+## Context
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)